views/partials/what_to_do.hbs

```diff
@@ -16,10 +16,11 @@
     <span class="bold">Subscribe</span> for alerts from Firefox Monitor to learn sooner about your compromised accounts.
   </li>
   <li>
-    <form action="/scan" method="post">
+    <form action="/scan" class="email-scan" method="post">
       <label class="medium">Scan another email address</label>
       <div class="input-group">
         <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email">
+        <input type="hidden" name="emailHash">
```

Comment:
Semi-related, but I submitted #147 to discuss why the form is duplicated in 2-3 places... and if it's possible to convert to a DRYer partial.